sites/footers/reports/a11y-1.json

```diff
@@ -0,0 +1,905 @@
+        "WCAG21Tech:general/G85",
+        "WCAG21Tech:general/G177",
+        "WCAG21Tech:client-side-script/SCR18",
+        "WCAG21Tech:client-side-script/SCR32",
+        "WCAG21Tech:general/G84"
+      ],
+      "earl:mode": "earl:automatic",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 3.3.4 Error Prevention (Legal, Financial, Data) (Level AA)",
+      "earl:test": "WCAG20:error-prevention-legal-financial-data)",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G164",
+        "WCAG21Tech:general/G98",
+        "WCAG21Tech:general/G155",
+        "WCAG21Tech:general/G99",
+        "WCAG21Tech:general/G168"
+      ],
+      "earl:mode": "earl:automatic",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 3.3.7 Accessible Authentication (Level A)",
+      "earl:test": "WCAG22:accessible-authentication",
+      "outcome": "Inapplicable",
+      "earl:outcome": "earl:inapplicable",
+      "notes": "",
+      "techniques": ["WCAG22Tech:general/G218"],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 3.3.8 Redundant entry (Level A)",
+      "earl:test": "WCAG22:redundant-entry",
+      "outcome": "Inapplicable",
+      "earl:outcome": "earl:inapplicable",
+      "notes": "",
+      "techniques": ["WCAG22Tech:general/G221"],
+      "earl:mode": "earl:manual",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 4.1.1 Parsing (Level A)",
+      "earl:test": "WCAG20:parsing",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G134",
+        "WCAG21Tech:general/G192",
+        "WCAG21Tech:html/H88",
+        "WCAG21Tech:failures/F70",
+        "WCAG21Tech:failures/F77"
+      ],
+      "earl:mode": "earl:automatic",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 4.1.2 Name, Role, Value (Level A)",
+      "earl:test": "WCAG20:name-role-value",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:aria/ARIA14",
+        "WCAG21Tech:aria/ARIA16",
+        "WCAG21Tech:general/G108",
+        "WCAG21Tech:general/G135",
+        "WCAG21Tech:general/G10",
+        "WCAG21Tech:failures/F59",
+        "WCAG21Tech:failures/F15",
+        "WCAG21Tech:failures/F20",
+        "WCAG21Tech:failures/F68",
+        "WCAG21Tech:failures/F79",
+        "WCAG21Tech:failures/F86",
+        "WCAG21Tech:failures/F89"
+      ],
+      "earl:mode": "earl:automatic",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 4.1.3 Status Messages (Level AA)",
+      "earl:test": "WCAG21:status-messages",
+      "outcome": "Pass",
+      "earl:outcome": "earl:pass",
```

Comment:
Do we have status messages in the footer, like content being dynamically update like the number of results items?

Reply:
As discussed with Brahim, this is what the automated tool has reported. Let's keep it as is.